### DISCUSSION TITLE
https://www.spsanderson.com/steveondata/posts/2024-10-17/

### DISCUSSION THREAD

**Discussion by giscus** [spsanderson/steveondata#13] [Comments]
# https://www.spsanderson.com/steveondata/posts/2024-10-17/

Discover how to efficiently loop through column names in R using various methods like for loops, lapply(), sapply(), and dplyr. Includes practical examples and best practices for beginner R programmers.

https://www.spsanderson.com/steveondata/posts/2024-10-17/

**Comment by phaecki** [comment]
I always look forward to new issues of your R articles and tutorials. They're written in a clear and practical way. However, I wonder if the barrier to liking or commenting is too high if I have to verify myself via the GitHub account first.

**Reply by spsanderson** [reply]
First, thank you for reading, much appreciatd. Secondly I understand, I did choose a high barrier to entry to prevent spam, maybe a discuss would have been better somehow but another thing is, I'm in github all day so it makes my life a bit easier.
